apis/nucleus/src/components/ActionsToolbarMore.jsx

```diff
@@ -52,18 +52,16 @@ const More = React.forwardRef(
           open={show}
           anchorEl={alignTo.current}
           getContentAnchorEl={null}
-          container={alignTo.current}
-          disablePortal
           hideBackdrop
           style={{ pointerEvents: 'none' }}
           transitionDuration={0}
           anchorOrigin={{
             vertical: 'bottom',
-            horizontal: 'right',
+            horizontal: 'center',
```

Comment:
@Caele Is this a breaking change or not?
If not ok, another possibility would be to add an option for the alignment.

Reply:
Tobias Had a point on since this is not a direct descendant of listbox (dom) tree, and since we can't align in exactly in center of it horizontally => then we should treat it as a normal dropdown which would open from bottom-left conner of it's anchor point like below 👇

<img width="250" alt="Screenshot 2023-01-12 at 16 13 20" src="https://user-images.githubusercontent.com/29652890/212104417-57e66e77-7ea8-484c-9f45-0b1def75bda6.png">